software/wger.yml

```diff
@@ -9,7 +9,7 @@ platforms:
 tags:
   - Miscellaneous
 source_code_url: https://github.com/wger-project/wger
-demo_url: https://wger.de/en/dashboard
+demo_url: https://github.com/wger-project/wger/tree/master/extras/docker/demo
```

Comment:
https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/PULL_REQUEST_TEMPLATE.md

> Demo links should only be used for interactive demos

The previous link was fine

Reply:
Weird. The link didn't seem to work when I checked it the other day.